src/plugins/tracing/nvtx/nvtx_backend.cpp

```
@@ -0,0 +1,173 @@
+            return "nvtx";
+        }
+
+    private:
+        const std::string domainName_;
+        nvtxDomainHandle_t domain_;
+    };
+
+} // namespace
+
+std::unique_ptr<TraceBackend>
+createNvtxBackend(const nixlTraceBackendInitParams &init_params) {
```

Cppcheck (2.21.0)

[style] The function 'createNvtxBackend' is never used.

(unusedFunction)